<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanel</class>
 <widget class="QWidget" name="ControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
  border: 1px solid skyblue;
  background-color: rgba(0,0,0,0);
  color: skyblue;
  padding: 5px;
}

QPushButton:disabled{
  color: #b5d9e8;
}

QPushButton:hover{

}

QPushButton:pressed{
color:#fff;

}</string>
  </property>
  <widget class="QPushButton" name="pushButton_bulbs">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>전구</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_hotBulbs">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>열전구</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_PDLC">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>PDLC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_waterPump">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>물펌프</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_doorMotor">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>220</y>
     <width>301</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>도어락 모터</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_doorLock">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>130</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="doorLock_OpenSW">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>370</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>도어락 열림감지 스위치</string>
   </property>
  </widget>
  <widget class="QLabel" name="doorLock_closeSW">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>430</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>도어락 닫힘감지 스위치</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_waterPumpSlider">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Qlabel_IPADDRESS">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>380</y>
     <width>371</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>Qlabel_IPADDRESS</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>90</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>순방향</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>90</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>역방향</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>70</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>순방향</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_4">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>110</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>역방향</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ignore</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
